Twig/Extension/CmfSeoExtension.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Ben Glassman <bglassman@gmail.com>
+ */
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Twig\Extension;
+
+use Symfony\Cmf\Bundle\SeoBundle\SeoPresentationInterface;
+
+class CmfSeoExtension extends \Twig_Extension
+{
+    /**
+     * @var SeoPresentationInterface
+     */
+    protected $seoPresentation;
+
+    /**
+     * @param SeoPresentationInterface $seoPresentation
+     */
+    public function __construct(SeoPresentationInterface $seoPresentation)
+    {
+        $this->seoPresentation = $seoPresentation;
+    }
+
+    /**
+     * @{inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('cmf_seo_update_seo_page',
```

Comment:
I'm not sure about the use of the word document here as it seems PHPCR specific and the object does not need to be a document, it might be a BlogPost from DoctrineORM or not even any kind of persisted object at all.